### PR TITLE
[AR-4938] watch asset docs

### PR DESCRIPTION
**Pull request by shaloo**
# PR

## Describe your changes

Added wallet_ watchAsset  in the usage guide for tracking new assets.

## Issue ticket number and link

- [AR-4938(https://team-1624093970686.atlassian.net/browse/AR-4938)

## Checklist before requesting a review

- [x] You have performed a self-review of your own code
- [x] You are using approved terminology
- [x] Your changes have been tested locally
- [x] Your code builds clean without any errors or warnings

### CODE REVIEW

File: usage.md [hunk 1]
```diff
@@ -266,4 +266,22 @@ interface SwitchEthereumChainParameter {
...
 
+### Track New Assets
+
+#### `wallet_watchAsset`
+
+This method is specified by [EIP-747](https://eips.ethereum.org/EIPS/eip-747).
+
+```ts
+params: {
+        type: 'ERC20',
```
Comment (makylfang) [inline usage.md:277]: Please fix the formatting and use provider.request like in other RPC requests specified

File: usage.md [hunk 2]
```diff
@@ -266,4 +266,22 @@ interface SwitchEthereumChainParameter {
...
+        options: {
+          address: '0xB983E01458529665007fF7E0CDdeCDB74B967Eb6',
+          symbol: 'FOO',
+          decimals: 18,
+          image: 'https://foo.io/token-image.svg',
+        },
+      },
+```
```
Comment (makylfang) [inline usage.md:285]: Replace with
```js
await provider.request({
  method: 'wallet_watchAsset',
  params: {
    type: 'ERC20',
    options: {
      address: '0xB983E01458529665007fF7E0CDdeCDB74B967Eb6',
      symbol: 'FOO',
      decimals: 18,
      image: 'https://foo.io/token-image.svg',
    },
  },
});
```

Comment (shaloo) [reply usage.md:285]: Yikes, I should have seen this.

**Comment by shaloo** [comment]
closing this PR and opening a new one.  This one is using hotfix branch as src for some reason.

**Pull request closed without merging**